packages/botonic-nlp/tests/tasks/ner/botonic-ner.test.ts

```diff
@@ -0,0 +1,157 @@
+import { join } from 'path'
+
+import { DatabaseManager } from '../../../src/embeddings/database-manager'
+import { DataLoader } from '../../../src/loaders/data-loader'
+import { Preprocessor } from '../../../src/preprocess/preprocessor'
+import { BotonicNer } from '../../../src/tasks/ner/botonic-ner'
+import { NerModelLoader } from '../../../src/tasks/ner/loaders/ner-model-loader'
+
+describe('Botonic NER', () => {
+  test('Loading model', async () => {
+    const ner = BotonicNer.from(
+      await NerModelLoader.from(join(__dirname, '..', '..', 'utils', 'model'))
```

Comment:
DRY: create a function testModelPath in tests/helpers (what you called tests/utils)

Reply:
What do you exactly mean? I know tests are a bit longer than usual but do you want me to create a function for each one so this way they are shorter? :thinking: